rep-0152.rst

```diff
@@ -0,0 +1,142 @@
+
+#. Abstract_
+#. Motivation_
+#. Reference_
+#. Operations_
+#. References_
+#. Copyright_
+
+Abstract
+========
+
+This REP specifies how we can organize and release packages without active maintainers.
+
+Motivation
+==========
+
+As the size of the ROS ecosystem grows, some packages will lose active maintainers.
+The main drawback of this situation is these packages do not get released into newer ROS distros which are released every year, even if they are commonly used (For example, USB camera drivers have not been released into Kinetic for more than half year [1]_ [2]_).
+Ideally, everyone wants to use a newer ROS distro which contains a set of packages almost equivalent to the set of packages in the previous ROS distro.
+
+It is understandable that the original maintainer of a package might not continue serving as an active maintainer due to starting a new project, moving institutions/companies, losing interest, and so on.
+Being a maintainer is not an easy job.
+It includes responding to incoming questions and issues, checking for proposed changes and deciding to add them to the main source tree, and other responsibilities.
+
+The idea behind this document is to organize a team of volunteers who can do ONLY the release job, which includes sending the source tree to the ROS Build Farm and acting on notification emails form the builds.
+The volunteers of the release maintainers group will enable the community to continue using the same set of packages from previous ROS distros with minimal community effort.
+To this end, we followed the Debian QA Group system and the Debian Developer's Reference ('Dealing with inactive and/or unreachable maintainers'[3]_), which is considered non-normative, but best practice.
+
+In ROS, we use 'Unmaintained' package status([4]_) and ROS Orphaned Packages Group([5]_), who do not have to take full responsibility for package maintenance jobs but are expected to take minimum responsibilities which mostly include releasing their packages into newer ROS distros.
+
+Reference
+=========
+
+REP-0140 [6]_ and REP-0141 [7]_ provide maintainer information in package manifest file and maintainer status in ROS distribution file.
+
+Maintenance Guide in ROSWiki [8]_ provides how to claim maintainership for orphaned packages.
+
+Operations
+==========
+
+1-A. Orphaning a package by maintainers
+---------------------------------------
+
+If you can no longer maintain your package.
+
+a. Set the package maintainer section in the ``package.xml`` file as follows.
+
+   ``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+b. Post a message to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_ indicating that package is now orphaned.
+   Wait for response from someone in the ROS Orphaned Package Maintainer team.
+
+c. Add admin permission of both source and release repository to who respond from in the ROS Orphaned Package Maintainers team.
```

Comment:
```suggestion
c. Add admin permission of both source and release repository to whomever responded from the ROS Orphaned Package Maintainers team.
```

Reply:
Also, what happens if the maintainer never gets a response?  That's not ideal, obviously, but in the case it does happen, what should the maintainer do?

Relatedly, how long should they wait for a response?  A week?  A month?

Reply:
If that happens, it means Ophaned team/framework is not working. I think that's is out of the scope of this document. how do you think?